src/opnsense/mvc/app/views/OPNsense/Core/license.volt

```diff
@@ -0,0 +1,21 @@
+<section class="col-xs-11">
+    <p><a href="https://opnsense.org/" target="_blank">OPNsense&reg;</a> is Copyright &copy; {{ product_copyright_years }} {{ product_copyright_owner }}<br>All rights reserved.</p>
```

Comment:
opnsense.org and "OPNsense" should be available as product_... view variables as well.

Reply:
I have changed it. After discussing it with Jos, I have also taken precautions to ensure that if someone changes the product name, it will still show our copyright correctly.

Reply:
makes sense, thanks 👍